src/components/StudyHeader/StudyHeader.js

```diff
@@ -25,7 +33,25 @@ const StudyHeader = ({
 
   return (
     <Container>
-      <Header as="h1">{studyName || 'Unknown study name'}</Header>
+      {newStudy && newStudy.length > 0 && (
+        <small
+          className={statusyMessage[currentStatus].class}
+          onClick={() => {
+            showModal(true);
+          }}
+        >
+          <Icon name={statusyMessage[currentStatus].icon} />
+          <span className="text-underline">
+            {statusyMessage[currentStatus].text}
+          </span>
+        </small>
+      )}
+      <Header
+        as="h1"
+        className={newStudy && newStudy.length > 0 ? 'mt-15' : ''}
+      >
+        {studyName || 'Unknown study name'}
```

Comment:
what case would we not have a study name? legacy studies?

Reply:
For legacy studies